lib/api/v3/work_packages/form/work_package_attribute_links_representer.rb

```diff
@@ -39,6 +39,23 @@ class WorkPackageAttributeLinksRepresenter < Roar::Decorator
           include Roar::Hypermedia
           include API::V3::Utilities::PathHelper
 
+          class << self
+            def create(work_package)
+              klass = Class.new(WorkPackageAttributeLinksRepresenter)
+              injector_class = ::API::V3::Utilities::CustomFieldInjector
+              linked_fields = work_package.available_custom_fields.select do |cf|
+                injector_class.linked_field?(cf)
+              end
+
+              injector = injector_class.new(klass)
+              linked_fields.each do |custom_field|
+                injector.inject_patchable_link_value(custom_field)
+              end
+
+              klass.new(work_package)
```

Comment:
This could probably be moved to the injector, because Version et al will also do something like this.